hybridse/src/vm/transform_request_mode_test.cc

```diff
@@ -466,6 +473,166 @@ TEST_P(TransformRequestModePassOptimizedTest, PassPassOptimizedTest) {
     PhysicalPlanCheck(catalog, in_out.first, in_out.second);
 }
 
+TEST_F(TransformRequestModePassOptimizedTest, SplitAggregationOptimizedTest) {
```

Comment:
Where is the physical plan node associated with w3?

Reply:
Updated. Disable the project list merge in the logical plan phase if long window exists.

Only doing the SplitPass seems that the w3 is still associated with the big window. Disabling the merge in the logical plan phase looks like an easy way.

Reply:
Well done!